### ISSUE TITLE
Temporary slot lockup after overlay interaction

### ISSUE THREAD

**Issue by ChloeDawn** [mezz/JustEnoughItems#2241]
**Mod version:** [1.16.4-7.6.1.71](https://www.curseforge.com/minecraft/mc-mods/jei/files/3192904)

After giving or deleting an item stack, inventory slots cannot be interacted with until an additional click has occurred.
*This occurs in both survival and creative mode, with only JEI present.*

**Steps to reproduce (giving):**
 - Spawn in item stack with give mode
 - Click on it (or another stack) to pick it up, it is not picked up
 - Click again, it is picked up

**Steps to reproduce (deleting):**
 - Delete item stack by dropping it onto overlay in cheat mode
 - Click on another stack in inventory, it is not picked up
 - Click again, it is picked up

**Comment by noobanidus** [comment]
I feel as though this may have been a regression from earlier 1.16 versions but I'm not certain. It's definitely something I can confirm is happening for me, as well as two other people that I happened to randomly poll at the time (KnightMiner & Flanks), with the former saying "I've gotten used to clicking twice when I delete items these days".

**Comment by maxiboy211** [comment]
Present in any version I've tried on 1.16.5. Currently using [jei-1.16.5-7.6.4.87](https://www.curseforge.com/minecraft/mc-mods/jei/files/3280247)
Tried with no other mods on Forge 36.1.0 and 36.1.13 and it still happens. Doesn't happen to me when spawning an item from JEI (right or left click/inventory or mouse pickup) but it always happens after deleting an item using JEI (dropping it on to the item grid on the right). Affects any inventory; player, chest, furnace and any modded inventory.
